src/main/java/de/uni_tuebingen/ub/nppm/servlet/AjaxServlet.java

```diff
@@ -282,6 +283,63 @@ private void newParentNode(HttpServletRequest request, HttpServletResponse respo
 
     }
 
+    private void doduplicate(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        try{
+             HttpSession session = request.getSession(true); 
+
+        int id = -1;
+        if (request.getParameter("duplicate") != null && request.getParameter("duplicate").equals(Language.getTextfield(session, "navigation", "Duplizieren"))) {
+            id = Integer.parseInt(request.getParameter("id"));
+
+            String sql = "INSERT INTO einzelbeleg (";
+            sql += "Belegnummer, Kontext_vor, Kontext, Kontext_nach, GeschlechtID, LebendVerstorbenID, EditionID, HandschriftID, ";
+            sql += "QuelleID, EditionKapitel, EditionSeite, QuelleGattungID, QuelleEchtheitID, QuelleDatierung, ";
+            sql += "UeberlieferungDatierung, Belegform, Griechisch, Diakritisch, KasusID, GrammatikGeschlechtID, ASWQuellenzitat, ";
+            sql += "Bemerkung, BearbeitungsstatusID, KommentarEthnie, KommentarAreal, KommentarVerwandtschaft, Eindeutig, ";
+            sql += "VonTag, VonMonat, VonJahr, VonJahrhundert, BisTag, BisMonat, BisJahr, BisJahrhundert, GenauigkeitVonTag, ";
+            sql += "GenauigkeitVonMonat, GenauigkeitVonJahr, GenauigkeitVonJahrhundert, DatierungUngewiss, KommentarDatierung, ";
+            sql += "LetzteAenderung, LetzteAenderungVon, Erstellt, ErstelltVon, GehoertGruppe, GenauigkeitBisTag, GenauigkeitBisMonat, ";
+            sql += "GenauigkeitBisJahr, GenauigkeitBisJahrhundert, GenauigkeitQuelleBisTag, GenauigkeitQuelleBisMonat, ";
+            sql += "GenauigkeitQuelleBisJahr, GenauigkeitQuelleBisJahrhundert, GenauigkeitQuelleVonTag, GenauigkeitQuelleVonMonat, ";
+            sql += "GenauigkeitQuelleVonJahr, GenauigkeitQuelleVonJahrhundert, QuelleBisTag, QuelleBisMonat, QuelleBisJahr, ";
+            sql += "QuelleBisJahrhundert, QuelleVonTag, QuelleVonMonat, QuelleVonJahr, QuelleVonJahrhundert, KommentarPerson, ";
+            sql += "MGHLemmaKorrigiert, KonventID, BeziehungGemeinschaftID, KritikID, KontextID, TitelText, pal_abgrenzung, ";
+            sql += "inh_abgrenzung, nr_in_strukt, seite, raster, schreiber, provenance_source, provenance_id) ";
```

Comment:
Die Provenance Informationen dürfen ebenfalls nicht dupliziert werden! Wenn jemand einen Datensatz dupliziert der ursprünglich aus DMP stammt, dann ist der duplizierte Datensatz ja in NeG erstellt worden